src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11020,7 +11020,7 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax unaryOrPrimar
                 // precedence level.  Examples include binary operator, assignment operators, range operators `..`, as
                 // well as `switch` and `with` clauses.
 
-                var (operatorTokenKind, operatorExpressionKind) = getOperatorTokenAndExpressionKind();
+                var (operatorTokenKind, operatorExpressionKind) = GetOperatorTokenAndExpressionKind();
```

Comment:
i would change the names as i recommended.  the pr will still be small in that case.